core/common/src/main/java/alluxio/util/CommonUtils.java

```diff
@@ -535,5 +536,41 @@ public static boolean isLocalHost(WorkerNetAddress address) {
     return address.getHost().equals(NetworkAddressUtils.getClientHostName());
   }
 
+  /**
+   * Closes the netty channel from outside the netty I/O thread.
+   *
+   * @param channel the netty channel
+   */
+  public static void closeChannel(final Channel channel) {
+    if (channel.isOpen())  {
+      try {
+        channel.eventLoop().submit(new Runnable() {
+          @Override
+          public void run() {
+            channel.close();
+          }
+        }).sync();
+      } catch (InterruptedException e) {
+        Thread.currentThread().interrupt();
+        throw new RuntimeException(e);
+      }
+    }
+  }
+
+  /**
+   * Closes the netty channel synchronously. Usually do not do this since this can take long time
+   * if the server is not responsive.
+   *
+   * @param channel the netty channel
+   */
+  public static void closeChannelSync(Channel channel) {
+    try {
+      channel.close().sync();
+    } catch (InterruptedException e) {
+      Thread.currentThread().interrupt();
+      throw new RuntimeException(e);
+    }
+  }
```

Comment:
I think it is ok since it is a simple util to close netty channel similar as how we close a closer. I can move it to another file if you prefer.